packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -220,7 +152,11 @@ const WalletTransfer = observer(({ is_wallet_name_visible, setIsWalletNameVisibl
                                     value={from_account}
                                 />
                             </div>
-                            <MessageList list={message_list} />
+                            <WalletTransferMessages
```

Comment:
**Note:** Moved the logic for generating the message list into `<WalletTransferMessages />` component.